ext/doc/Cassandra.php

```diff
@@ -322,7 +322,12 @@
     /**
      * Current version of the extension.
      */
-    const VERSION = '1.0.0-beta';
+    const VERSION = '1.1';
+
+    /**
+     * Version of the cpp-driver the extension is compiled against.
+     */
+    const CPP_DRIVER_VERSION = '2.2.2';
```

Comment:
Nevermind ... This looks like it is just for the documentation stubs and not what is [built into the library](https://github.com/datastax/php-driver/pull/58/files#diff-d3776f8e62027b140044da1533b29c4fR82)